Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tracing-opentelemetry"
-version = "0.24.0"
+version = "0.25.0"
```

Comment:
It's a bit confusing that the version of tracing-opentelemetry would be 0.25 while the rest of opentelemetry crates would be at 0.24. Is that intentional?

Reply:
It's been like this for a while now. This is just how semver works out sometimes.